config/example.yaml

```diff
@@ -1,30 +1,31 @@
 api:
   name: staff-fee-privilege
-  endpointUri: https://api.oregonstate.edu/v1
+  endpointUri: ${ENDPOINTURI}
 
 server:
   basePath: /api/v1
-  port: 8080
-  adminPort: 8081
+  port: ${PORT}
```

Comment:
Could you mention these `${FOO}` variables can be set via env vars?

Reply:
Done.